testpilot/frontend/static-src/app/views/experiment-list-page.js

```diff
@@ -1,38 +1,20 @@
-import app from 'ampersand-app';
-import ExperimentRowView from './experiment-row-view';
-
+import ExperimentListView from './experiment-list-view';
 import PageView from './page-view';
 
+import template from '../templates/experiment-list-page';
+
 export default PageView.extend({
-  pageTitle: 'Test Pilot - Experiments',
+  pageTitle: 'Firefox Test Pilot',
   pageTitleL10nID: 'pageTitleExperimentListPage',
-  template: `<div class="page" >
-               <section data-hook="experiment-list-page">
-                 <header data-hook="main-header"></header>
-                 <ul id="idea-card-list" class="experiments"></ul>
-               </section>
-               <div data-hook="main-footer" class="vertical-flex-container"></div>
-             </div>`,
+  template: template,
 
   render() {
-    // TODO: this is not awesome
-    document.body.id = document.body._id;
-    document.body.id = 'list-view';
-
     PageView.prototype.render.apply(this, arguments);
-
-    // render the experiment list into the page
-    this.experimentList = this.renderCollection(
-      app.experiments,
-      ExperimentRowView,
-      this.query('.experiments')
-    );
+    this.renderSubview(new ExperimentListView({loggedIn: true}),
```

Comment:
Should `loggedIn` be hardcoded to `true` here?

**DISCLAIMER:** I have no clue what I'm talking about. I just wanted to confirm if we should check the logged in state versus hardcoding.